sdks/java/io/hadoop/jdk1.8-tests/src/test/resources/kubernetes/elasticsearch/es-client-node.yaml

```diff
@@ -29,6 +29,10 @@ spec:
         component: elasticsearch
         role: client
       annotations:
+      # Elasticsearch uses a hybrid mmapfs / niofs directory by default to store its indices. 
+      # The default operating system limits on mmap counts is likely to be too low, which may result
+      # in out of memory exceptions. Therefore, the need to increase virtual memory 
+      # vm.max_map_count for large amount of data in the pod initialization annotation.
         pod.beta.kubernetes.io/init-containers: '[
           {
           "name": "sysctl",
```

Comment:
From definition on Kubernetes page, Init Containers, which are specialized Containers that run before app Containers and can contain utilities or setup scripts not present in an app image
http://stackoverflow.com/questions/41491290/set-sysctl-key-in-kubernetes-pod-using-kops-docker-1-11, here we can see that sysctl is used in init containers. When we remove this command , from our script, we actually see the pod status in crashloopback status, this setting is required to have the pod in the running state. So when we applied the virtual memory command for sysctl in init-containers, our pods successfully start without showing error.